girder/models/item.py

```diff
@@ -438,6 +443,15 @@ def stream():
                 yield json.dumps(doc['meta'], default=str)
             yield (os.path.join(path, metadataFile), stream)
 
+    def _mimeFilter(self, file, mimeFilter):
+        """
+        Returns whether or not the given file should be passed through the given
+        MIME filter. If no MIME filter is specified, all files are allowed.
+        """
+        if not mimeFilter:
```

Comment:
I think `filter` is acceptable nomenclature for whitelist semantics; in fact, it's similar to the semantics of the python builtin of the same name. If I were making a blacklist instead, I'd probably call it a mask rather than a filter.